lib/BleFingerprint/util.h

```diff
@@ -11,6 +11,7 @@ static BLEUUID sonosUUID((uint16_t)0xFE07);
 static BLEUUID itagUUID((uint16_t)0xffe0);
 static BLEUUID miThermUUID(uint16_t(0x181A));
 static BLEUUID trackrUUID((uint16_t)0x0F3E);
+static BLEUUID vanmoofUUID("6acc5540-e631-4069-944d-b8ca7598ad50");
```

Comment:
```suggestion
static BLEUUID vanmoofUUID(0x6acc5540, 0xe631, 0x4069, 0x944db8ca7598ad50);
```
This saves memory as opposed to a string

Reply:
Good to know, thanks!